machineconfiguration/install.go

```diff
@@ -0,0 +1,26 @@
+package machineconfiguration
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	machineconfigurationv1 "github.com/openshift/api/machineconfiguration/v1"
+)
+
+const (
+	GroupName = "machineconfiguration.openshift.io"
```

Comment:
this group already exists and this is a straight move?

Reply:
yes: https://github.com/openshift/machine-config-operator/blob/master/pkg/apis/machineconfiguration.openshift.io/install.go#L9